generator/generator.go

```diff
@@ -110,6 +110,16 @@ func execute(command string, force bool) error {
 		args = args[1:]
 	}
 
+	// Templates will often include a helm:generate header such as the following:
```

Comment:
Should still use `helm:generate`.  The "language" of the templated needs to not change at all... we want to be backwards and forwards compatible.